shader/draw_leaf.wgsl

```diff
@@ -74,184 +72,198 @@ fn main(
     // Two barriers can be eliminated if we use separate shared arrays
     // for prefix and intra-workgroup prefix sum.
     workgroupBarrier();
-    var m = sh_scratch[0];
-    workgroupBarrier();
-    let tag_word = read_draw_tag_from_scene(ix);
-    agg = map_draw_tag(tag_word);
-    sh_scratch[local_id.x] = agg;
-    for (var i = 0u; i < firstTrailingBit(WG_SIZE); i += 1u) {
+    var prefix = sh_scratch[0];
+
+    let num_blocks_total = (config.n_drawobj + (WG_SIZE - 1u)) / WG_SIZE;
```

Comment:
nit: You could remove the parentheses around `WG_SIZE - 1u`.

Reply:
Basically the reason for these parentheses was that I didn't trust constant evaluation to understand the associativity of addition. But I think it's better to trust the compiler here, and have the cleanest expression of programmer intent.